<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeRemoteWidget</class>
 <widget class="QWidget" name="ChangeRemoteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>&amp;Url:</string>
     </property>
     <property name="buddy">
      <cstring>urlLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="urlLineEdit">
     <property name="placeholderText">
      <string>wss://example.com/app/</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="secretCheckBox">
     <property name="text">
      <string>&amp;Secret:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="secretLineEdit">
     <property name="placeholderText">
      <string>Uncheck box to keep current</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="clearDataLabel">
     <property name="text">
      <string>&amp;Clear Data:</string>
     </property>
     <property name="buddy">
      <cstring>clearDataCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="clearDataCheckBox">
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>secretCheckBox</tabstop>
  <tabstop>secretLineEdit</tabstop>
  <tabstop>clearDataCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>secretCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>secretLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
